clients/src/main/java/org/apache/kafka/clients/consumer/internals/AbstractCoordinator.java

```diff
@@ -431,11 +438,11 @@ public void handle(SyncGroupResponse syncResponse,
                            RequestFuture<ByteBuffer> future) {
             Errors error = Errors.forCode(syncResponse.errorCode());
             if (error == Errors.NONE) {
-                log.info("Successfully joined group {} with generation {}", groupId, generation);
```

Comment:
I relocated this message to the join callback inside `ensureActiveGroup`. I think I was annoyed at having to grab the lock just to print this message and we were already holding inside the `onSuccess()` method.